lib/modules/hearing_test/screens/hearing_test_history_results/hearing_test_history_results.dart

```diff
@@ -0,0 +1,223 @@
+                                                        onPressed:
+                                                            () => Navigator.of(
+                                                              context,
+                                                            ).pop(true),
+                                                        child: Text(
+                                                          'Delete',
+                                                          style: TextStyle(
+                                                            color: Colors.red,
+                                                          ),
+                                                        ),
+                                                      ),
+                                                    ],
+                                                  ),
+                                            );
+                                            if (confirm == true) {
+                                              await file.delete();
+                                              (context as Element)
+                                                  .markNeedsBuild();
+                                            }
+                                          },
+                                        ),
+                                        onTap: () {
+                                          selectedIndexNotifier.value =
+                                              selectedIndex == index
+                                                  ? null
+                                                  : index;
+                                        },
+                                      ),
+                                      if (selectedIndex == index)
+                                        Container(
+                                          margin: const EdgeInsets.symmetric(
+                                            horizontal: 20,
+                                            vertical: 20,
+                                          ),
+                                          height: 420,
+                                          decoration: BoxDecoration(
+                                            border: Border.all(
+                                              color: Colors.grey.shade300,
+                                            ),
+                                            borderRadius: BorderRadius.circular(
+                                              10,
+                                            ),
+                                          ),
+                                          padding: const EdgeInsets.fromLTRB(
+                                            10,
+                                            20,
+                                            20,
+                                            10,
+                                          ),
```

Comment:
I think using the `.only()` method makes future development more straightforward and developer-friendly.

```suggestion
                                          padding: const EdgeInsets.only(
                                            left: 10,
                                            top: 20,
                                            right: 20,
                                            bottom: 10,
                                          ),
```

Reply:
We can leave it as it is right now, but I disagree. As the name suggests `fromLTRB - Left Top Right Bottom`. `only` is used when you want to add padding only on specific side.